apis/io-engine/protobuf/v1/common.proto

```diff
@@ -7,3 +7,17 @@ enum ShareProtocol {
   NVMF = 1; // NVMe over Fabrics (TCP)
   ISCSI = 2; // iSCSI
 }
+
+message MayastorFeatures {
+  // NVMe ANA enablement status.
+  bool asymmetricNamespaceAccess = 1;
+  // LVM backend presence and enablement status.
```

Comment:
@tiagolobocastro , shouldnt this be snake case?

Reply:
Existing type before I moved it was like that if you see, so I kept in.
Doesn't make a difference to generated code though, this is just on the proto.
I can change to snake though if you like?